WordPress/src/main/res/values/strings.xml

```diff
@@ -1341,6 +1341,7 @@
     <string name="blog_name_invalid">Invalid site address</string>
     <string name="blog_title_invalid">Invalid site title</string>
     <string name="username_invalid">Invalid username</string>
+    <string name="checking_username">Checking username</string>
```

Comment:
I know you're trying to be consistent with the `username_invalid` entry above this, but how about "user name" instead of "username"?